plugins/netskope/icon_netskope/actions/patch_url_list_by_id/schema.py

```diff
@@ -0,0 +1,152 @@
+# GENERATED BY KOMAND SDK - DO NOT EDIT
+import insightconnect_plugin_runtime
+import json
+
+
+class Component:
+    DESCRIPTION = "Update the name, URLs, and/or type of a URL list object"
+
+
+class Input:
+    ACTION = "action"
+    ID = "id"
+    NAME = "name"
+    TYPE = "type"
+    URLS = "urls"
+    
+
+class Output:
+    DATA = "data"
+    ID = "id"
+    MODIFY_BY = "modify_by"
+    MODIFY_TIME = "modify_time"
+    MODIFY_TYPE = "modify_type"
+    NAME = "name"
+    PENDING = "pending"
+    
+
+class PatchUrlListByIdInput(insightconnect_plugin_runtime.Input):
+    schema = json.loads("""
+   {
+  "type": "object",
+  "title": "Variables",
+  "properties": {
+    "action": {
+      "type": "string",
+      "title": "Action",
+      "description": "Replace or append to current URLs",
+      "enum": [
+        "replace",
+        "append"
+      ],
+      "order": 2
+    },
+    "id": {
+      "type": "integer",
+      "title": "ID",
+      "description": "ID of the URL list",
+      "order": 1
+    },
+    "name": {
+      "type": "string",
+      "title": "Name",
+      "description": "Name of replaced URL list",
+      "order": 3
+    },
+    "type": {
+      "type": "string",
+      "title": "URL List Category",
+      "description": "Category of URL list",
+      "enum": [
+        "exact",
+        "regex"
+      ],
+      "order": 5
+    },
+    "urls": {
+      "type": "array",
+      "title": "URLs",
+      "description": "List of URLs",
+      "items": {
+        "type": "string"
+      },
+      "order": 4
+    }
+  },
+  "required": [
+    "action",
```

Comment:
this is a patch call, why all the fields are mandatory. Can you please double check.

Reply:
Every input parameter is required there - according to the API documentation